cudax/include/cuda/experimental/__launch/configuration.cuh

```diff
@@ -53,11 +54,12 @@ enum class launch_option_kind
 struct option_not_found
 {};
 
+// TODO could this return references, so we don't copy the options
```

Comment:
Not if it is host device because we do not want to have dangling references across memory boundaries